libevm/VMOpt.cpp

```diff
@@ -49,20 +49,22 @@ void VM::initMetrics()
 	done = true;
 }
 
-//
-// init interpreter on entry
-//
-
+/// Init interpreter on entry.
 void VM::initEntry()
 {
-	m_bounce = &VM::interpretCases;	
+	m_bounce = &VM::interpretCases;
+
+	// Copy and extend code by 32 bytes to handle virtual push data.
```

Comment:
Also I don't see where this actually copies the code. Furthermore, doesn't `m_codeSpace.resize(extendedSize)` imply a reserve of (at least) the same size?